pkg/gadgets/run/tracer/tracer.go

```diff
@@ -376,6 +376,46 @@ func (t *Tracer) attachProgram(gadgetCtx gadgets.GadgetContext, p *ebpf.ProgramS
 		case strings.HasPrefix(p.SectionName, "kretprobe/"):
 			logger.Debugf("Attaching kretprobe %q to %q", p.Name, p.AttachTo)
 			return link.Kretprobe(p.AttachTo, prog, nil)
+		case strings.HasPrefix(p.SectionName, "uprobe/"):
```

Comment:
Could the code be factorized a bit with something like:
```suggestion
		case strings.HasPrefix(p.SectionName, "uprobe/") || strings.HasPrefix(p.SectionName, "uretprobe/"):
```

Only the lines with `ex.Uprobe` or `ex.Uretprobe` differ.